### PR TITLE
Update eZ Publish to eZ Platform

### PR DESCRIPTION
**Pull request by andrerom**
A bit overdue update, so updated several parts of the text a bit while at it.

Review/input welcome, just comment inline.

Todo:
- [x] Re add logo

### CODE REVIEW

File: projects/ezplatform.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: eZ Platform
+url: https://ezplatform.com/
+dependencies:
+    - https://github.com/ezsystems/ezplatform/blob/master/composer.json
+    - https://github.com/ezsystems/ezplatform-ee/blob/master/composer.json
+description: |
+    The eZ Platform, as of v2 a pure full stack Symfony CMS, is the foundation
+    for the eZ Platform  Enterprise Edition, which is built for business-critical
```
Comment (natanael89) [inline projects/ezplatform.yml]: There are two spaces between "Platform" and "Enterprise".

**Comment by andrerom** [comment]
> Since deleting ezpublish.png, an eZ Platform icon will be added, right?

yes, re added it straight from the source :)

review ping @rolandbenedetti

**Comment by javiereguiluz** [comment]
@andrerom thanks for updating this. However, the new logo file is not valid. We need a square logo (160 x 160 px is the size we use for new projects). Something like this: http://symfony.com/uploads/projects/ezpublish.png  If this doesn't look OK, you can remove the "eZ" letters (or leave only the "eZ" letters). I know it's a compromise, but as you can see at http://symfony.com/projects all projects display a simple image instead of their full logo. Thanks!

Here is an empty logo file template image in case it's useful for you:

symfony-project-logo-blank

**Comment by andrerom** [comment]
Ok @javiereguiluz. Was unsure about that since many projects have different size, I'll resize based on raw svg file and re-upload.

**Comment by andrerom** [comment]
@javiereguiluz Updated

**Pull request merged**